pkg/server/chaosd/attack.go

```diff
@@ -51,7 +51,10 @@ func (s *Server) ExecuteAttack(attackType AttackType, options core.AttackConfig,
 		return
 	}
 
-	uid = uuid.New().String()
+	uid = options.GetUID()
```

Comment:
Is it because we can directly specify a UID?

Reply:
Yes